### PR TITLE
'Find' module update and 'Apply' module build

### PR DESCRIPTION
**Pull request by garethwilliamsdesign**
### Trello card

[Trello 4741](https://trello.com/c/cZKO47mQ)

### Context

We received feedback from the adviser conference that, ”Users struggle to find the links to Find and Apply.” So, we wish to improve the visibility of the existing 'Find' module and create a new 'Apply' module in order to remedy this.

### Changes proposed in this pull request

Update of the 'Find' module and creation of a new 'Apply' module.

### Guidance to review

Review the two pages onto which these modules have been added (/train-to-be-a-teacher/how-to-apply-for-teacher-training and /train-to-be-a-teacher/if-you-have-a-degree) and check rendering on a variety of browsers and screen sizes.

### CODE REVIEW

File: app/webpacker/styles/category.scss [hunk 1]
```diff
@@ -174,25 +174,39 @@
 }
 
 .find-your-course {
-  margin: auto;
+  margin: 4em auto 0;
   max-width: $content-max-width;
-  border-top: 1px solid $grey-mid;
-  padding-top: 3em;
+  background-color: #F1F1F1;
```
Comment (philspray) [inline app/webpacker/styles/category.scss:179]: think this has to be lower case?

**Review by philspray — COMMENTED** [review]
This looks good to me from a content point of view.

**Comment by EllieNodder** [comment]
Is it me, or is this rendering a bit strange on the /train-to-be-a-teacher/if-you-dont-have-a-degree page? It's coming after the related content links on there and there's no gap between the module and the line break? @garethwilliamsdesign 

image

this also happens on the following pages:

/what-is-a-pgce
/funding-and-support/salaried-teacher-training
/funding-and-support/scholarships-and-bursaries
/funding-and-support/tuition-fee-and-maintenance-loans
/is-teaching-right-for-me/qualifications-you-need-to-teach

also, shall we add the apply module to the other apply pages (personal statements and references?) or do you reckon that's overkill? @philspray

**Comment by philspray** [comment]
I agree @EllieNodder, it makes sense to add the apply module to personal statements and references.

**Comment by garethwilliamsdesign** [comment]
@EllieNodder, well spotted! I should’ve mentioned that I’ve not actually applied the changes across all the pages yet. Unfortunately, I’ll have to update every page that features these modules in order to rearrange the content and ensure that the modules appear before the related content when viewed on mobile. I’ll do this once we’re happy with the changes on the two pages you’ve already seen, although I won’t be able to publish anything until Pavel’s back.

**Comment by EllieNodder** [comment]
> @EllieNodder, well spotted! I should’ve mentioned that I’ve not actually applied the changes across all the pages yet. Unfortunately, I’ll have to update every page that features these modules in order to rearrange the content and ensure that the modules appear before the related content when viewed on mobile. I’ll do this once we’re happy with the changes on the two pages you’ve already seen, although I won’t be able to publish anything until Pavel’s back.

ah cool, makes sense to me! happy with the /train-to-be-a-teacher/if-you-have-a-degree and /train-to-be-a-teacher/how-to-apply-for-teacher-training changes :)

File: app/webpacker/styles/category.scss (hunk 1 above)
Comment (garethwilliamsdesign) [reply app/webpacker/styles/category.scss]: Good spot. Sorted.

**Review by plisovin — APPROVED** [review]
LGTM

**Pull request merged**